packages/workflow/src/Interfaces.ts

```diff
@@ -573,6 +577,12 @@ namespace ExecuteFunctions {
 			fallbackValue?: any,
```

Comment:
might as well fix this 
```suggestion
			fallbackValue?: boolean,
```